tests/h/services/search_index/_queue_test.py

```diff
@@ -96,6 +96,51 @@ def annotation_ids(self, annotations):
         return [annotation.id for annotation in annotations]
 
 
+class TestAddUsersAnnotations:
+    @pytest.mark.parametrize("force,schedule_in", [(True, 42), (False, None)])
+    def test_it(self, userid, users_annotations, db_session, queue, force, schedule_in):
+        queue.add_users_annotations(
+            userid, tag="test_tag", force=force, schedule_in=schedule_in
+        )
+
+        assert (
+            db_session.query(Job).all()
+            == Any.list.containing(
+                [
+                    Any.instance_of(Job).with_attrs(
+                        {
+                            "tag": "test_tag",
+                            "name": "sync_annotation",
+                            "scheduled_at": Any.instance_of(datetime_.datetime),
+                            "priority": 100,
+                            "kwargs": {
+                                "annotation_id": str(
+                                    uuid.UUID(
+                                        URLSafeUUID.url_safe_to_hex(annotation.id)
+                                    )
+                                ),
+                                "force": force,
+                            },
+                        }
+                    )
+                    for annotation in users_annotations
+                ]
+            ).only()
+        )
```

Comment:
If / when we have a unified path for queuing things, it'll let us simplify these tests a bit, as we won't need to test the whole SQL construction again.